package.json

```diff
@@ -100,8 +100,10 @@
     "react-dom": "16.8.2",
     "redux-mock-store": "1.5.3",
     "remote-redux-devtools": "0.5.16",
+    "shelljs": "0.8.3",
```

Comment:
Introduce `shelljs` package to easily run shell command in javascript and easily do control flow.